### ISSUE TITLE
Unable to rearrange favorites on the panel

### ISSUE THREAD

**Issue by ShrirajHegde** [home-sweet-gnome/dash-to-panel#1351]
Error in journalctl during drag and drop : `clutter_input_device_ungrab: assertion 'CLUTTER_IS_INPUT_DEVICE (device)' failed`

Errors on gnome-shell restart 

```
Can't update stage views actor MetaWindowGroup is on because it needs an allocation.
Can't update stage views actor MetaWindowActorX11 is on because it needs an allocation.
Can't update stage views actor MetaSurfaceActorX11 is on because it needs an allocation.
Can't update stage views actor MetaWindowActorX11 is on because it needs an allocation.
Can't update stage views actor MetaSurfaceActorX11 is on because it needs an allocation.
```
**Describe the bug**
Unable to rearrange favorite apps on the panel

**Linux distribution and version**
Fedora 34

**GNOME Shell version**
GNOME Shell 40.1

**Dash-to-Panel version**
2a8b2ffec54403012448501b2e9e25f057383542 , 3fb7bdb83b69478341ec1568b31ef6829d004715

**Where was Dash-to-Panel installed from?**
Github (`make install`)

**Additional info**

This bug happens only on mouse, works fine on touchpad

**Comment by Robbendebiene** [comment]
You actually can arrange them if you just hold your mouse down a little bit (but still not too long because then the context menu will open). This is probably due to the following gnome "bug" and may be automatically fixed with the next GNOME release.
See: https://gitlab.gnome.org/GNOME/gnome-shell/-/issues/4287 and https://gitlab.gnome.org/GNOME/gnome-shell/-/merge_requests/1866

**Comment by ShrirajHegde** [comment]
This has been fixed already! , I am on Gnome 40.2
